index.html

```diff
@@ -91,7 +91,7 @@ <h4 class="section3-heading">Proffesional Art Printing Data</h4>
           <li class="section3-item1">HTML</li>
           <li class="section3-item2">Bootstrap</li>
           <li class="section3-item3">Ruby</li>
-        </ul>
+        </ul> -->
```

Comment:
- [ ] Kindly, add the bar line, at the end of the about section  

- Figma Design:

![gg](https://user-images.githubusercontent.com/57562869/150011381-b81dfd6a-9df3-4795-a6e2-128dce768bca.png)

- Your Design:

![g3](https://user-images.githubusercontent.com/57562869/150011541-b25f9330-c266-4805-87b7-5e68fa605085.png)